### ISSUE TITLE
能不能打包成dockers部署

### ISSUE THREAD

**Issue by wbext** [PKU-YuanGroup/MagicTime#2]

**Comment by inFaaa** [comment]
Thanks for your interest. We will consider your opinions and launch a Docker version in the future. We are currently deploying the model Demo to huggingface via Gradio.

**Comment by SHYuanBest** [comment]
Thanks for your interest. Currently, MagicTime only supports text-to-video generation and does not currently support image-to-video generation. We will consider your opinions in the future.

**Comment by truedat101** [comment]
Agree, after seeing #23 , dockers can act as a sort of "CI" server test run to quickly verify that all instructions in the shell scripts work.
